caffe2/opt/converter.cc

```diff
@@ -101,6 +101,14 @@ std::vector<int> getKernelShape(std::map<std::string, caffe2::Argument> argMap)
   return kernelShape;
 }
 
+int getGroup(std::map<std::string, caffe2::Argument> argMap) {
```

Comment:
pass by value?

Reply:
suppose to give a default value to group field.
BTW, can we give default values to other fields, like dilations, pads... ?

Reply:
I don't understand. Why can't we use `const std::map<std::string, caffe2::Argument>&` here?

Reply:
Looks like we have different code base. Will fix it soon.